doc/reference/modules/nhibernate_caches.xml

```diff
@@ -153,7 +153,15 @@
 				they are destroyed with the SessionFactory instance; so you must keep them alive as long as you need
 				them.
 			</para>
+			<para>
+				The second level cache requires the use of transactions, be it through transaction scopes or NHibernate
+				transactions. Interacting with the data store without an explicit transaction is discouraged, and will
+				not allow the second level cache to work as intended.
+			</para>
 		</formalpara>
+		<para>
+			See also <xref linkend="performance-cache" />.
+		</para>
```

Comment:
We have two separated section documenting second level cache, each with valuable information not available in the other. It would be better to merge them. At least I add a cross link here. (The other side already does it.)